includes/class-wc-gateway-cardknox.php

```diff
@@ -419,9 +418,12 @@ public function payment_scripts()
 
         wp_enqueue_script('cardknox', 'https://cdn.cardknox.com/ifields/2.15.2401.3101/ifields.min.js', '', '1.0.0', false);
         wp_enqueue_script('woocommerce_cardknox', plugins_url('assets/js/cardknox' . $suffix . '.js', WC_CARDKNOX_MAIN_FILE), array('jquery-payment'), filemtime(plugin_dir_path(dirname(__FILE__)) . 'assets/js/cardknox' . $suffix . '.js'), true);
+        
+
+        $encrypted_key = $this->token_key;
```

Comment:
Is this key encrypted? Who is decrypting it?

Reply:
Hello @mstein-ck 
No, this key is not encrypted.it's just a variable name.

Reply:
@mstein-ck Changed the variable name.

![image](https://github.com/user-attachments/assets/dc2228ba-37a4-4b7c-bd51-8b845dc06c4e)